src/components/FlexibleForm/Control.js

```diff
@@ -219,37 +219,111 @@ export const Control = memo(props => {
 
   const attribs = getAttributes();
 
+  const renderChildren = () => children.map((cfg, i) => (
+    <Control
+      key={`control-${i}`}
+      repeatableIndex={cfg.repeatable ? i : repeatableIndex}
+      staticNamespace={staticNamespace}
+      editableNamespace={editableNamespace}
+      sectionNamespace={sectionNamespace}
+      intl={intl}
+      styles={styles}
+      referenceTables={referenceTables}
+      setReferenceTables={setReferenceTables}
+      initialFields={initialFields}
+      injectedProps={injectedProps}
+      commonSections={commonSections}
+      record={attribs.record}
+      {...cfg}
+    />
+  ));
+
+  const renderMCL = contentData => {
+    const Cmp = getControl('MultiColumnList');
+
+    const {
+      legend,
+      visibleColumns,
+      columnWidths,
+      columnMapping,
+      mclColumnPaths,
+    } = props;
+
+    const data = isEmpty(mclColumnPaths) ? contentData : contentData.map(row => {
+      let currentRow = {};
+
+      visibleColumns.forEach(col => {
+        const colPaths = mclColumnPaths[col];
+        const fieldName = get(row, colPaths.namePath, col);
+        const fieldValue = get(row, colPaths.valuePath, <stripesComponents.NoValue />);
+
+        currentRow = {
+          ...currentRow,
+          [fieldName]: fieldValue,
+        };
+      });
+
+      return currentRow;
+    });
+
+    let mapping = {};
+
+    Object.keys(columnMapping).forEach(key => {
+      const mapValue = columnMapping[key];
+
+      mapping = {
+        ...mapping,
+        [key]: isTranslationId(mapValue) ? <FormattedMessage id={mapValue} /> : mapValue,
+      };
+    });
+
+    return (
+      <>
+        <div>
+          <strong>{isTranslationId(legend) ? <FormattedMessage id={legend} /> : legend}</strong>
+        </div>
+        <Cmp
+          contentData={data}
+          visibleColumns={visibleColumns}
+          columnWidths={columnWidths}
+          columnMapping={mapping}
+        />
+        <br />
```

Comment:
Why is the `<br>` tag needed here?

Reply:
It's needed to simply separate MCL from the next section

Reply:
Isn't the MCL component block-level element?

Reply:
Yes, but it requires styles adjustments which is not wanted here